examples/manufacturers/Brother/Printers/MFC-J6920DW/MFC-J6920DW.type.d3.yaml

```
@@ -3,7 +3,8 @@ credentialSubject:
   # unique identifiers
   id: aa92ebae-4928-4456-b597-cbd7657e9e83
   #aliases is an URI
-  aliases: '["www.manysecured.net/aa92ebae-4928-4456-b597-cbd7657e9e83", "www.brother.com/aa92ebae-4928-4456-b597-cbd7657e9e83"]'
+  aliases: ["www.manysecured.net/aa92ebae-4928-4456-b597-cbd7657e9e83",
+  "www.brother.com/aa92ebae-4928-4456-b597-cbd7657e9e83"]
```

We should probably encourage hyphenated lists like with other fields
```suggestion
  aliases:
    - "www.manysecured.net/aa92ebae-4928-4456-b597-cbd7657e9e83"
    - "www.brother.com/aa92ebae-4928-4456-b597-cbd7657e9e83"
```